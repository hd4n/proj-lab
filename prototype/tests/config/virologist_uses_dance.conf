#effekt_nev id duration
@effects
da d1 3

#agent_nev id effekt_id
@agents
vi vi1 d1

#kod_nev id agent_id aminocost nucleocost
@codes
co cd1 vi1 0 0

#mezo_nev id [tartalom]
@fields
em e1
em e2

#mezo_id1 mezo_id2
#ketiranyu kapcsolat
@neighbors
e1 e2

#id akt_mezo [item_id1;item_id2...] [material_id1;material_id2...] [effect_id1;effect_id2...]
@virologists
v1 e1 null null null
v2 e2 null null null