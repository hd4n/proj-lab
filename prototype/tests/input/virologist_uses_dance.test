load virologist_uses_dance.conf
getstats v2
craft v1 cd1 #kell?
useagent v1 v2 d1
getstats v2
