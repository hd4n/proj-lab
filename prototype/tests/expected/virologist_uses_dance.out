Load: sikeres

stats v2:
fi_e2
eq_null
ma_null
ef_null
co_null

craft: c1 cd1 vi1 #kell?

useagent:
v1+v2
dance

stats v2:
fi_e2
eq_null
ma_null
ef_d1
co_null
